solidity/core/contracts/validator-manager/MultisigValidatorManager.sol

```diff
@@ -251,6 +260,11 @@ abstract contract MultisigValidatorManager is Ownable {
      * @param _domain The domain to hash.
      */
     function _domainHash(uint32 _domain) internal pure returns (bytes32) {
-        return keccak256(abi.encodePacked(_domain, "ABACUS"));
+        if (VERSION > 0) {
```

Comment:
Suggest we update the natspec explaining what `domainHash` means now. Originally, the `domain` acted as both a chain ID *and* a domain separator via `domainHash`. Now that we've decoupled those roles, `_domainHash` is a confusing name, and we should clarify with an @dev